tools/hex.sh

```diff
@@ -27,7 +27,7 @@ send_hex_udp () {
 }
 
 send_hex_tcp () {
-  echo "$hex" | xxd -r -p | nc "$host" "$port"
+  echo "$hex" | xxd -r -p | nc -N "$host" "$port"
```

Comment:
What does `-N` mean? It doesn't seem to be universally supported, so I would recommend removing.